### PR TITLE
js-algorand-sdk v1.13.1

### PR DESCRIPTION
**Pull request by onetechnical**
# 1.13.1

## Added:

- Add app creator to dryrun request (#499)

## Changed:

- Adding note to use bigint to help debug when amount/assetAmount is large (#501)

## Fixed:

- Fix JSON decoding (#502)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# 1.13.1
+
+## Added:
+
+- Add app creator to dryrun request (#499)
+
+Changed:
```
Comment (jasonpaulos) [inline CHANGELOG.md:7]:
```suggestion
## Changed:
```

**Review by jasonpaulos — COMMENTED** [review]
Also needs to update the readme with new version and hash

**Comment by onetechnical** [comment]
> Also needs to update the readme with new version and hash

Thanks, was pre-mature. Should be all set now.

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ or
 ```html
 <script
   src="https://cdn.jsdelivr.net/npm/algosdk@1.13.0/dist/browser/algosdk.min.js"
```
Comment (jasonpaulos) [inline README.md:33]: Both of the links should say 1.13.1 now

```suggestion
  src="https://cdn.jsdelivr.net/npm/algosdk@1.13.1/dist/browser/algosdk.min.js"
```

Comment (onetechnical) [reply README.md]: *sigh* good catch!

**Review by jasonpaulos — APPROVED** [review]

**Review by egieseke — APPROVED** [review]

**Pull request merged**